### ISSUE TITLE
AttributeError: 'EncDecSpeakerLabelModel' object has no attribute 'get_label'

### ISSUE THREAD

**Issue by egorsmkv** [NVIDIA/NeMo#5577]
**Describe the bug**

I cannot get the language of a file with AmberNet

**Steps/Code to reproduce bug**

Code:

```python
import nemo 
from nemo.collections.asr.models import EncDecSpeakerLabelModel

model = EncDecSpeakerLabelModel.from_pretrained('langid_ambernet')

print('NeMO:', nemo.__version__)

print('AmberNet method:', model.get_label)
```

Result:

```
...

NeMO: 1.13.0

Traceback (most recent call last):
  File "/Users/yehorsmoliakov/Work/egorsmkv/asr-corpus-creator/rep.py", line 8, in <module>
    print('AmberNet method:', model.get_label)
  File "/Users/yehorsmoliakov/.local/share/virtualenvs/asr-corpus-creator-Sp_hMA78/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1265, in __getattr__
    raise AttributeError("'{}' object has no attribute '{}'".format(
AttributeError: 'EncDecSpeakerLabelModel' object has no attribute 'get_label'
```

**Expected behavior**

Info about the method `get_label`

**Comment by egorsmkv** [comment]
Can I help in resolving of this issue?

**Comment by titu1994** [comment]
You have to use the main branch, this hasn't been released yet in r1.13

**Comment by titu1994** [comment]
@fayejf is this being released in r1.14? If so, you'll need to wait just a week or two

**Comment by skanda1005** [comment]
I am facing the same issue but it says 'EncDecSpeakerLabelModel' object has no attribute 'batch_inference'. I am on the main branch only, Please help!
Thanks

**Comment by egorsmkv** [comment]
1.14 release happened 4 days ago

As I understand, this was not fixed it this release, correct?

**Comment by titu1994** [comment]
Apparently it did not make the 1.14 release. Two ways to go if you need the functionality ASAP - use main branch commit hash, or subclass and add in the function.

**Comment by egorsmkv** [comment]
Thanks!

**Comment by egorsmkv** [comment]
@titu1994 hi, is it going to be fixed in the 1.15 release?

**Comment by titu1994** [comment]
Yes it should be there
